src/handlers/artist.rs

```diff
@@ -296,6 +296,16 @@ pub fn handler(key: Key, app: &mut App) {
           handle_recommend_event_on_selected_block(app);
         }
       }
+      Key::Char('w') => match artist.artist_selected_block {
+        ArtistBlock::Albums => app.current_user_saved_album_add(ActiveBlock::ArtistBlock),
+        ArtistBlock::RelatedArtists => app.user_follow_artists(ActiveBlock::ArtistBlock),
+        _ => (),
+      },
+      Key::Char('D') => match artist.artist_selected_block {
+        ArtistBlock::Albums => app.current_user_saved_album_delete(ActiveBlock::ArtistBlock),
```

Comment:
As a side note, these destructive actions should now be handled after a confirmation dialog.

This can be done in another PR though.